vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.kt

```diff
@@ -1041,78 +1045,125 @@ class VectorSettingsPreferencesFragment : PreferenceFragmentCompat(), SharedPref
      * Update the password.
      */
     private fun onPasswordUpdateClick() {
+        activity?.let { activity ->
+            val view: ViewGroup = activity.layoutInflater.inflate(R.layout.dialog_change_password, null) as ViewGroup
 
-        val thisActivity = activity
-        thisActivity?.let { activity ->
-            val view = activity.layoutInflater.inflate(R.layout.dialog_change_password, null)
-            val oldPasswordText = view.findViewById<EditText>(R.id.change_password_old_pwd_text)
-            val newPasswordText = view.findViewById<EditText>(R.id.change_password_new_pwd_text)
-            val confirmNewPasswordText = view.findViewById<EditText>(R.id.change_password_confirm_new_pwd_text)
+            val oldPasswordTil: TextInputLayout = view.findViewById(R.id.change_password_old_pwd_til)
```

Comment:
OK so by inlining error this is what was needed showing the error in TextInputLayout only . That seems better informing about error .

Reply:
Yes, sorry if it wasn't clear :)

Reply:
NP ,.next time I would explicitly ask .